### PR TITLE
feat:added servictype page

### PR DESCRIPTION
**Pull request by ebin-davis**

### CODE REVIEW

File: src/components/adminComponents/ServiceTypeComponent.jsx
Comment (JaseelaAlimahamed) [inline src/components/adminComponents/ServiceTypeComponent.jsx:1]: make a folder inside adminComponent to set this file

File: src/pages/adminPages/ServiceType.jsx
Comment (JaseelaAlimahamed) [inline src/pages/adminPages/ServiceType.jsx:1]: dont make this much code iin pages

**Review by JaseelaAlimahamed — CHANGES_REQUESTED** [review]
dont make too much lines in page , make it component and call it in pages,

**Comment by ebin-davis** [comment]
completed the work

**Pull request merged**